pyscript.core/tests/integration/test_py_terminal.py

```diff
@@ -1,12 +1,30 @@
 import time
 
+import pytest
 from playwright.sync_api import expect
 
-from .support import PyScriptTest, skip_worker
+from .support import PageErrors, PyScriptTest, skip_worker
 
 
 class TestPyTerminal(PyScriptTest):
-    @skip_worker("FIXME: the auto worker dance removes terminal")
+    def test_single_terminal(self):
```

Comment:
Shouldn't it be `test_multiple_terminals`?

Reply:
right ... done + added tests for worker case with also input